readme.txt

```diff
@@ -110,7 +110,17 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
-= 1.2 =
+= 1.2.1 =
+
+**Release Date:** December 17, 2020.
+
+* New feature: status check in the editor
```

Comment:
Nit: Calling this a feature may confuse users who might look for its UI. How about "Automated plugin compatibility status check."?

Reply:
It doesn't have much to do with plugin compatibility though.

Maybe `New feature: automated site environment compatibility check`?